pkg/netstate/netstate.go

```diff
@@ -83,6 +107,17 @@ func (s *Server) List(ctx context.Context, req *pb.ListRequest) (*pb.ListRespons
 	s.logger.Debug("entering netstate list")
 
 	pathKeys, err := s.DB.List()
+
```

Comment:
uh oh, now not checking the err here. probably the list should happen after anyway

Reply:
I will enable errcheck on the linter